Jenkinsfile

```
@@ -73,8 +73,8 @@ pipeline {
                defaultValue: getPriority(),
                description: 'Priority of this build.  DO NOT USE WITHOUT PERMISSION.')
         string(name: 'TestTag',
-               defaultValue: "daily_regression",
-               description: 'Test-tag to use for this run (i.e pr, daily_regression, full_regression, etc.')
+               defaultValue: "",
+               description: 'Test-tag to use for this run (i.e. pr, daily_regression, full_regression, etc.')
```

I was thinking of the other typo.
```suggestion
               description: 'Test-tag to use for this run (i.e. pr, daily_regression, full_regression, etc.)')
```